File: Python_Virtual_World/Wolf.py
from MyVector import MyVector
from Animal import Animal


class Wolf(Animal):


    STRENGTH = 9
    INITIATIVE = 5


    def __init__(self, position : MyVector):
        super().__init__(position, Wolf.STRENGTH, Wolf.INITIATIVE)


    def draw(self) -> str:

        return "red"


    def __str__(self):
        return "Wolf"

File: Java_Virtual_World/src/Gameplay/Animals/Wolf.java
package Gameplay.Animals;

import Gameplay.Organism;

public class Wolf extends Animal{
    public static final int STRENGTH = 9;
    public static final int INITIATIVE = 5;

    public Wolf(int x, int y){
        super(x ,y , STRENGTH, INITIATIVE);
    }
    @Override
    public String Draw(){
        return "🐺";
    }
    @Override
    public Organism Copy(){
        return new Wolf(x,y);
    }
    @Override
    public String toString(){
        return "Wolf";
    }
}
